src/main/java/io/pravega/connector/boomi/PravegaWriteOperation.java

```diff
@@ -76,7 +76,7 @@ protected void executeUpdate(UpdateRequest request, OperationResponse response)
                     response.addResult(input, OperationStatus.APPLICATION_ERROR, STATUS_CODE, STATUS_MESSAGE, null);
                 } else {
                     //Pravega only support to write event as a string. So we have to convert input stream to string
-                    String message = inputStreamToUtf8String(input.getData());
+                    String message = inputToUtf8String(input);
```

Comment:
on line 71, modify the comment so it is accurate - we are now using SizeLimitedUpdateOperation, so explain how the data size will be checked here and in SizeLimitedUpdateOperation, and what sizes are acceptable